tests/utils/v2Data.js

```diff
@@ -0,0 +1,27 @@
+const uuid = require('uuid');
+const { UtapiMetric } = require('../../libV2/models');
+
+function range(n, step) {
+    const vals = [...Array(n).keys()];
+    if (step) {
+        return vals.map(i => i * step);
+    }
+    return vals;
+}
```

Comment:
For efficiency reasons, I think a generator will work best for a range method ( copies python basically ).
```suggestion
function* range(count, step=1) {
    for (let i = 0; i < count; i+=step) {
      yield i;
    }
}
```

Reply:
Oh if you meant for non-integers then I think this would work.
```suggestion
function* range(count, step=1) {
    let i = 0;
    while (i < count * step) {
      yield i;
      i += step;
    }
}
```